packages/esm-patient-vitals-app/src/vitals/paginated-vitals.scss

```
@@ -13,12 +19,22 @@
   }
 }
 
-.criticallyLow, .criticallyHigh {
+.criticallyHigh span, .criticallyLow span {
```

And the same in similar places...

```suggestion
.criticallyHigh > span, .criticallyLow > span {
```

---

@ibacher I have updated the PR to meet this. Thanks!